multi-agent/lib/mas/core/auth/credentials/models.py

```
@@ -0,0 +1,68 @@
+
+    Lookup key: ``(user_id, server_identifier)``.
+    """
+    id: str = Field(default_factory=lambda: uuid4().hex)
+    user_id: str
+    server_identifier: str = Field(
+        default="",
+        description="Auth server issuer URL (e.g. https://accounts.google.com)",
+    )
+    access_token: str
+    refresh_token: Optional[str] = None
+    token_type: str = "Bearer"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "token_type"

(S105)